### PR TITLE
Pair Programming Contact me From

### PR DESCRIPTION
**Pull request by GeekyHacks**
In this pull request, I did the following:
-Pair program with my partner @sahlminkok to finalize the contact me form.
- Set up the contact me form with the right semantics.
- Set up buttons with more interactive styling.
- Link the form with formspree.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,12 +1,15 @@
 * {
```
Comment (huseyinbiyik) [inline styles.css:1]: Kindly follow the design guideline on Figma and fix the font size and font-family issue. To use a font family, first you need to import or download to your repository. You can use the import method on this [link](https://fonts.google.com/specimen/Inter#styles) for Inter font for this section. 
Screenshot 2023-04-04 at 9 23 21 PM
Screenshot 2023-04-04 at 9 24 31 PM

File: index.html [hunk 2]
```diff
@@ -244,25 +234,41 @@ <h2 class="contact-me-header">
...
         </h2>
-        <div class="input-form">
-          <input type="text" placeholder="Full name" /><input
+        <form
+          autocomplete="off"
+          action="https://formspree.io/f/xdovwjar"
+          method="post"
+          class="input-form"
+        >
+          <input
```
Comment (huseyinbiyik) [inline index.html:243]: Good job with using Flexbox when positioning the elements 👍  Let's follow the template and add the same amount of space between input areas and button 👍 
Screenshot 2023-04-04 at 9 33 27 PM
Screenshot 2023-04-04 at 9 33 50 PM

File: index.html [hunk 3]
```diff
@@ -244,25 +234,41 @@ <h2 class="contact-me-header">
...
+            name="message"
           ></textarea>
-        </div>
-        <div class="green-btn">
-          <a href="https://github.com/GeekyHacks">
-            <button type="button" title="press">Get in touch</button>
-          </a>
-        </div>
-        <div class="contact-email">geekyhacks22@gmail.com</div>
+          <button name="submit" class="green-btn" type="submit" title="press">
```
Comment (huseyinbiyik) [inline index.html:267]: Please follow the design guideline and add the Hover and Active state styling for your form submit button. 👍 The main goal of this project is to code the given design perfectly.  
Screenshot 2023-04-04 at 9 39 25 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Form submission works well 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝

## Required Changes ♻️
-  Please make sure you use imperative tense and capitalize the first word for your commits. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (GeekyHacks) [reply styles.css:1]: I did import the suggested font but it looks different from whats in the template
Kazam_screenshot_00083

Comment (huseyinbiyik) [reply styles.css:1]: Hello @GeekyHacks. You are importing 'Lato' font. Check the GIF below.
kkk

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi, @GeekyHacks,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**